husky_components/datagrid/decorators/table-view.js

```diff
@@ -1092,8 +1099,10 @@ define(function() {
         editableInputFocusoutHandler: function(event) {
             if (!!this.isFocusoutHandlerEnabled) {
                 this.sandbox.dom.stopPropagation(event);
-                var recordId = this.sandbox.dom.data(this.sandbox.dom.parents(event.currentTarget, '.' + constants.rowClass), 'id');
-                this.editRow(recordId);
+
+                var e = jQuery.Event('keypress');
```

Comment:
@3rfan Actually Apart from the fact that a global variable is used here I would think this approach is better than handling another state manually... The only advantage I see in your solution is that we could save all remaining changes once instead immediately after every change, which we don't do currently... /cc @chirimoya